static/new_home/style.css

```diff
@@ -12,6 +12,12 @@ body
   padding: 0; 
 }
 
+section {
+  box-sizing: border-box;
```

Comment:
This property is already applicable everywhere with Bootstrap. You don't need to explicitly add it.